app.js

```diff
@@ -11,56 +11,74 @@ const totalEl = document.getElementById('total');
 const lossesEl = document.getElementById('losses');
 const winsEl = document.getElementById('wins');
 
-
 let correctGuesses = 0;
 let totalGuesses = 0;
 
 shedButton.addEventListener('click', () => {
     // get a random item to call the 'correct spot'
-
+    const correctSpot = getRandomHidingSpot();
+    handleGuess('shed', correctSpot);
     // call the handleGuess function with the correct parameters (the user's guess and the "correct" hiding place) to do DOM work
+    // console.log(correctSpot);
+    // handleGuess('shed');
 });
 
 treeButton.addEventListener('click', () => {
     // get a random item to call the 'correct spot'
-
+    const correctSpot = getRandomHidingSpot();
+    handleGuess('tree', correctSpot);
     // call the handleGuess function with the correct parameters (the user's guess and the "correct" hiding place) to do DOM work
+    console.log(correctSpot);
```

Comment:
In the future, you may consider commenting or deleting console.log() once you've verified the functionality.